MediaBrowser.Providers/MediaInfo/FFProbeProvider.cs

```diff
@@ -59,11 +59,12 @@ public class FFProbeProvider : ICustomMetadataProvider<Episode>,
             IChapterManager chapterManager,
             ILibraryManager libraryManager,
             IFileSystem fileSystem,
+            ILoggerFactory loggerFactory,
             NamingOptions namingOptions)
         {
             _logger = logger;
```

Comment:
You can get rid of one dependency in the constructor by using the factory here as well
```suggestion
            _logger = loggerFactory.CreateLogger<FFProbeProvider>();
```